pages/game_page.py

```diff
@@ -1,3 +1,17 @@
 import streamlit as st
+import random
 
-st.title("GAME PAGE")
\ No newline at end of file
+st.title("Build a Scientist")
+
+# Display scientist  
+# st.image("images/scientist-beaker-test-tube-animated-f.gif")
+# Display scientist with equipment 
+
+game_words = ['Genetic','Histology','Sequence','Clinical Trial','Mutation','Genome','Oncology']
+
+def game_word_generator(word):
+    st.write("".join("_" * len(word)))
```

Comment:
Streamlit uses markdown under the hood for string formatting, which turns any sequence of dashes into a solid line like you saw. From this stack overflow I found that adding `` around the string fixed it. Also added spaces in between the dashes

```suggestion
    st.write(f"`{' '.join(['_'] * len(word))}`")
```